knotx-core/knotx-engine-view/src/test/resources/test-no-fragments.html

```diff
@@ -1,14 +0,0 @@
-<html>
```

Comment:
Why you removed those test htmls ? there were tests using this, isn't ? or it's moved to splitter ?